views/pages/funding/programmes.njk

```diff
@@ -9,14 +9,16 @@
 {% set socialImage = heroImage %}
 
 {% block content %}
+    {% set pageAccent = 'pink' %}
     <main role="main">
         {{ hero(
             titleText = title,
             image = heroImage,
-            accent = 'green'
```

Comment:
Is this right? The colour change? Currently has this:

![image](https://user-images.githubusercontent.com/394376/38550378-4108f3d4-3cae-11e8-9ef4-724f5d81f2f3.png)

Reply:
I've made a guess that it's a mistake that the page title is green but the programmes are pink.